File: Chapter 2 List & Array/Array/Q26 Remove Duplicates from Sorted Array/Solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.empty()){
            return 0;
        }
        
        int index = 0;
        for(int i = 0; i < nums.size(); i++){
            if(nums[i] != nums[index]){
                nums[++index] = nums[i];
            }
        }
        return index + 1;
    }
};

File: Chapter 2 List & Array/Array/Q26 Remove Duplicates from Sorted Array/Solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length == 0){
            return 0;
        }
        
        int unique = 0;
        for(int i = 0; i < nums.length; i++){
            if(nums[i] != nums[unique]){
                unique++;
                nums[unique] = nums[i];
            }
        }
        
        return unique + 1;
    }
}